lib/src/components/card_rounded.dart

```diff
@@ -49,20 +66,33 @@ class CardRounded extends StatelessWidget {
       child: Container(
         height: height,
         width: width,
-        padding: const EdgeInsets.symmetric(
-          vertical: 12,
-          horizontal: 12,
-        ),
+        padding: padding ??
+            const EdgeInsets.symmetric(
+              vertical: 12,
+              horizontal: 12,
+            ),
+        child: child,
       ),
     );
 
     if (onClose != null) {
-      const offset = -5.0;
-      _closeButton = Positioned(
-        right: offset,
-        top: offset,
-        child: _CloseButton(onTap: onClose),
-      );
+      if (closeButtonAlignment == Alignment.topRight) {
+        _closeButton = Positioned(
+          right: _defaultOffset,
+          top: _defaultOffset,
+          child: _CloseButton(onTap: onClose, iconData: customButtonIcon),
+        );
+      } else if (closeButtonAlignment == Alignment.bottomRight) {
+        _closeButton = Positioned(
+          right: _defaultOffset,
+          bottom: _defaultOffset,
+          child: _CloseButton(onTap: onClose, iconData: customButtonIcon),
+        );
+      } else {
+        throw Exception(
+          'Card supports only Alignment.topRight and Alignment.bottomRight',
+        );
```

Comment:
This is more of a programmer error, not a runtime exception, is it?

Perhaps let's replace it with an `assert`, what do you think?